src/bootstrap/lib.rs

```diff
@@ -108,9 +111,29 @@ pub mod util;
 #[cfg(windows)]
 mod job;
 
-#[cfg(not(windows))]
+#[cfg(unix)]
 mod job {
-    pub unsafe fn setup() {}
+    use libc;
+
+    //apparently glibc defines their own enum for this parameter, in a different type
+    #[cfg(not(any(target_env = "musl", target_env = "musleabi", target_env = "musleabihf",
+                  target_os = "emscripten", target_arch = "mips", target_arch = "mipsel")))]
+    const PRIO_PGRP: libc::c_uint = libc::PRIO_PGRP as libc::c_uint;
```

Comment:
The re-declaration is necessary because of [how `setpriority` is defined][setpriority] for glibc platforms:

[setpriority]: https://github.com/rust-lang/libc/blob/a285387a9e8fb010d310193f87a98800c94b3084/src/unix/notbsd/linux/other/mod.rs#L495

```rust
pub fn setpriority(which: ::__priority_which_t, who: ::id_t,
                                   prio: ::c_int) -> ::c_int;
```

Where `__priority_which_t` is [at the top of the file][priority_t]:

[priority_t]: https://github.com/rust-lang/libc/blob/a285387a9e8fb010d310193f87a98800c94b3084/src/unix/notbsd/linux/other/mod.rs#L4

```rust
pub type __priority_which_t = ::c_uint;
```

Without this re-declaration, I get a type error when calling `setpriority` on `x86_64-unknown-linux-gnu`.